setupRadio.js

```diff
@@ -0,0 +1,30 @@
+function setupRadio(x, y, radioOptions, verticalFlag, defaultOption) {
+  var BUTTON_WIDTH = 30;

```

Comment:
Could you add a quick comment to just note this is in pixels (should be obvious, but y'know)